src/OrbitGl/TimeGraph.cpp

```diff
@@ -877,7 +882,9 @@ void TimeGraph::DoDraw(orbit_gl::PrimitiveAssembler& primitive_assembler,
   // TODO(http://b/217719000): We are drawing boxes in margin positions because some elements are
   // being drawn partially outside the TrackContainer space. This hack is needed until we assure
   // that no element is drawn outside of its parent's area.
-  DrawMarginsBetweenChildren(primitive_assembler);
+  if (layout_->GetDrawTimeGraphMasks()) {
```

Comment:
Why we need this?

Reply:
For debugging

Reply:
What I mean is that it's a useful debugging feature, the code is meant to be merged.